Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -45,12 +45,16 @@ import com.ibm.jzos.ZFile;
  * Version 6 - 2024-03
  *  - Added support to write IBM Wazi Deploy Application Manifest file
  *      
+ * Version 7 - 2024-04
+ *  - Added support to SBOM files
+ *      
  ************************************************************************************/
 
 // start create & publish package
 @Field Properties props = null
 def scriptDir = new File(getClass().protectionDomain.codeSource.location.path).parent
 @Field def wdManifestGeneratorUtilities = loadScript(new File("${scriptDir}/utilities/WaziDeployManifestGenerator.groovy"))
+@Field def sbomUtilities = loadScript(new File("${scriptDir}/utilities/sbomGenerator.groovy"))
```

Comment:
@M-DLB You need to conditionally load the `sbomUtilities`.

I am getting a ClassNotFound Exception when not providing the cycloneDX libraries on the class path.

We expect customers using the script w/o using the SBOM feature.

Reply:
A snippet from zAppBuild to conditionally load the utility script.
```
@Field def resolverUtils
// Conditionally load the ResolverUtilities.groovy which require at least DBB 1.1.2
if (buildUtils.useSearchPathAPI()) {
	resolverUtils = loadScript(new File("${props.zAppBuildDir}/utilities/ResolverUtilities.groovy"))
	}
```